src/Seq.App.Slack/SlackReactor.cs

```diff
@@ -67,14 +67,29 @@ public class SlackReactor : Reactor, ISubscribeTo<LogEventData>
             HelpText = "If a property when converted to a string is longer than this number it will be truncated")]
         public int? MaxPropertyLength { get; set; } = null;
 
+        [SeqAppSetting(
+            DisplayName = "Included properties",
+            IsOptional = true,
+            HelpText = "Comma separated list (no spaces) of properties to include as attachments.  The default is to include all properties.")]
+        public string IncludedProperties { get; set; }
+
         private EventTypeSuppressions _suppressions;
         private static readonly IImmutableList<string> SpecialProperties = ImmutableList.Create("Id", "Host");
-        private static SlackApi _slackApi;
+        private static ISlackApi _slackApi;
```

Comment:
I think it's become fragile marking this field as `static`, especially because tests may assign it via the constructor in parallel; perhaps we could remove the modifier, now?

Reply:
Agreed, I've removed the `static` modifier.